packages/terra-menu/src/Menu.jsx

```diff
@@ -0,0 +1,215 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Popup from 'terra-popup';
+import SlideGroup from 'terra-slide-group';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import MenuItem from './MenuItem';
+import MenuItemGroup from './MenuItemGroup';
+import MenuDivider from './MenuDivider';
+import MenuContent from './_MenuContent';
+import MenuWidths from './_MenuWidths';
+import MenuUtils from './_MenuUtils';
+import styles from './Menu.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * List of Menu.Item(s)/Menu.ItemGroup(s)/Menu.Divider(s) to be displayed as content within the Menu.
+   */
+  children: PropTypes.node.isRequired,
```

Comment:
I initially saw this while reading down through the PR. I actually thing node is good.